packages/lib/WebDavApi.js

```diff
@@ -441,7 +442,16 @@ class WebDavApi {
 				throw newError(`${message} (Exception ${code})`, response.status);
 			}
 
-			throw newError('Unknown error 2', response.status);
+			let message = 'Unknown error 2';
+			if (response.status === 401 || response.status === 403) {
+				if (!authToken) {
+					message = _('Access denied: Empty password. You may have to re-enter your WebDAV credentials.');
+				} else {
+					message = _('Access denied: Invalid credentials');
```

Comment:
In general user-facing errors should explicitly tell the user what needs to be done. So in that case, "Access denied: Please check your username and password"